concent_api/core/tasks.py

```diff
@@ -56,12 +57,15 @@ def upload_finished(subtask_id: str) -> None:
         # If subtask is past the deadline, processes the timeout.
         if parse_datetime_to_timestamp(subtask.next_deadline) < get_current_utc_timestamp():
             # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
-            bankster.finalize_payment(
+            finalize_deposit_claim(
                 subtask_id=subtask_id,
                 concent_use_case=ConcentUseCase.ADDITIONAL_VERIFICATION,
```

Comment:
This Celery task has `@atomic`. `finalize_deposit_claim()` won't be able to commit its transaction.

Reply:
:+1: done